### ISSUE TITLE
Feature request: allow for non-repeating headers in multi-page tables

### ISSUE THREAD

**Issue by simonthelwall** [ropensci/tabulizer#99]
Some tables spread across multiple pages, but the header of the table only appears on the first page. 
This means that `extract_tables(file, pages, output = "data.frame")` returns dataframes the names of the data frame contain data rather than the true names. 

It would be great if `extract_tables` had an option `repeating_header =  FALSE` so that each returned dataframe had the same column names.

**Comment by pm321** [comment]
I agree, the default behaviour of extract_tables in assuming that the first row comprises headings leads to unsatisfactory results. I often need to extract tables from pdf format where there are no headings; furthermore I would like to be able to set my own headings; so a function along the lines of col_names = c("var_1", "var_2", "var_n" ) which allows the addition of user defined column headings would be a great improvement.

**Comment by tedmoorman** [comment]
I agree that this is a very important feature. It looks like the development of this package is dead, which is too bad.

**Comment by tedmoorman** [comment]
It looks like you can at least say `header = FALSE` to avoid names of the data frame with data rather than true column names.
